src/apps/Calculators/AxByEqualsP/util/getAxByScale.js

```diff
@@ -0,0 +1,43 @@
+const getAxByScale = (firstSteps, firstCents, secondSteps, secondCents) => {
+  let stepDifferences = [];
+
+  if (firstSteps > secondSteps) {
+    for (let i = 0; i < firstSteps; i++) {
+      stepDifferences.push(firstCents);
+    }
+    const disperser = Math.ceil(firstSteps / secondSteps);
+    for (let i = disperser; i <= disperser * secondSteps; i += disperser) {
+      stepDifferences.splice([i], 0, secondCents);
+    }
+  }
+
+  if (firstSteps < secondSteps) {
+    for (let i = 0; i < secondSteps; i++) {
+      stepDifferences.push(secondCents);
+    }
+    const disperser = Math.ceil(secondSteps / firstSteps);
+    for (let i = disperser; i <= disperser * firstSteps; i += disperser) {
+      stepDifferences.splice([i], 0, firstCents);
+    }
+  }
+
+  if (firstSteps === secondSteps) {
+    for (let i = 0; i < firstSteps; i++) {
+      stepDifferences.push(firstCents);
+      stepDifferences.push(secondCents);
+    }
+  }
+
+  let scale = [];
```

Comment:
Array (or any kind of object) declarations should always be `const` unless you're actually reassigning them (ie doing `scale = [...]` later). Mutating objects (eg by reassigning an object property or calling `push` on an array) is fine as the variable is just storing a reference to the location of the object in memory - it's only by changing where that reference points to that results in a `TypeError`.